### DISCUSSION TITLE
Github Education can't verify my University of colorado boulder academic affiliation !

### DISCUSSION THREAD

**Discussion by akkefa** [community/community#60024] [GitHub Education]
### Hi everyone,

Hi,

I am doing MS in Data Science from the University of colorado boulder remotely. I provide the admission email as proof but GitHub education did not accept it.

The reason the mentioned this university is not located in your country.

```
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘Other’ option when submitting a GitHub Education support ticket.

Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible.

You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote. Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘Other’ option when submitting a GitHub Education support ticket.
```

**Comment by samus-aran** [comment]
Hi @akkefa 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

**Comment by akkefa** [comment]
@samus-aran Thank you for replying back. 🙏🙏🙏

 I am still trying its not working due to my current location. My university, the University of Colorado Boulder, is located in the US, so I am not in the US.
 
 Please help me sort out this issue.
